### DISCUSSION TITLE
‘like’  sql syntax error in version 0.12.1

### DISCUSSION THREAD

**Discussion by luanshidujun** [apache/iotdb#3940] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37785928/132671341-564944cb-8e45-44a9-bcfa-0445bcd27313.png)

**Comment by HTHou** [comment, marked as answer]
0.12.1 didn’t support like yet. This feature is in 0.12.2.

**Reply by luanshidujun** [reply]
excuse me,  Where do I download the release version( 0.12.2)?

**Reply by HTHou** [reply]
It will be available in few days.

**Reply by HTHou** [reply]
Hi, 0.12.2 is released. You can enjoy the new feature now!
https://iotdb.apache.org/Download/
